cluster-autoscaler/cloudprovider/aws/aws_wrapper.go

```diff
@@ -85,7 +85,7 @@ func (m *awsWrapper) getManagedNodegroupInfo(nodegroupName string, clusterName s
 	}
 
 	if r.Nodegroup.CapacityType != nil && len(*r.Nodegroup.CapacityType) > 0 {
-		labels["capacityType"] = *r.Nodegroup.CapacityType
+		labels["eks.amazonaws.com/capacityType"] = *r.Nodegroup.CapacityType
```

Comment:
Should we abstract this label to a const variable and use it everywhere so that it is easy to change if the describe API starts using the correct label in the future?

Reply:
Hm, would you mind rephrasing the question? The NodeGroupDescribe API for capacity type only returns a value (in this case, either `SPOT` or `ON_DEMAND`), and this change here only adds the prefix that's used on the node, so that the label map cache that gets pre-populated at startup actually matches the labels added on the node. The API call isn't really using an "incorrect" label, it was just being indexed wrong on the map 😅

Reply:
That being said, using a variable to save the "label key" does make sense! But I'd preferably like to do that as part of a separate PR so that it encompasses all the labels used by the "GetNodeGroupInfo()" function, as a larger refactor.

Reply:
> The NodeGroupDescribe API for capacity type only returns a value (in this case, either SPOT or ON_DEMAND)

You are right. Please ignore my comment.

Reply:
> But I'd preferably like to do that as part of a separate PR so that it encompasses all the labels used by the getManagedNodegroupInfo() function, as a refactor of sorts. 😄

Sounds good to me :+1: